GitAutomation/GitAutomation.psd1

```diff
@@ -148,8 +148,12 @@
             # ReleaseNotes of this module
 
             ReleaseNotes = @'
-* Added `Get-GitConfiguration` function for getting Git configuration values.
-* Added `Remove-GitConfiguration` function for removing/unsetting Git configuration values.
+* Fixed: `ConvertTo-GitFullPath` doesn't properly encode the returned Uri string. (thanks @thorbenw)
+  * as described in Issue #32
+* Fixed: `Merge-GitCommit`'.when there are conflicts'-test fails spuriously due to different line terminators when run in Windows-based environments. (thanks @thorbenw)

```

Comment:
Remove this line, this is an internal change to a test that doesn't change anything with the actual module and therefore shouldn't be in release notes.

Reply:
Done.